CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ retired at the same time.
 
 * The response format of the [callcaching/diff](https://github.com/broadinstitute/cromwell#get-apiworkflowsversioncallcachingdiff) endpoint has been updated.
 
+### Cromwell Server
+
+* Cromwell now attempts to gracefully shutdown when running in server mode and receiving a `SIGINT` or `SIGTERM` signal. This includes waiting for all pending Database writes before exiting.
```

Comment:
I'd add in something making it clear that `SIGINT` == `^C` and `SIGTERM` == `kill`, most people are going to think in those terms